src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -40,7 +40,7 @@ namespace Microsoft.IdentityModel.JsonWebTokens
     /// <summary>
     /// A <see cref="SecurityToken"/> designed for representing a JSON Web Token (JWT). 
```

Comment:
Should we add a remark to this class and the handler that this is newer and preferred for use than other Jwt classes in System namespace?

Reply:
This seems like its a source of confusion for people so I definitely think it's a good idea to add this in! It's also probably a good idea to add this on the old Jwt token and handler classes as well.

Did both in this [commit](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1320/commits/6cc69b1500a8507cde6c582e7314e248a27dc5a2).

Reply:
Isn't the old supposed to be marked as obsolete? with details saying to use the new one?